lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,21 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str) or len(text) < 1:
+        return None
+
+    tokens = ''
+    word = ''
```

Comment:
Is this variable really needed?

Reply:
Done (changed the function to avoid this)